source/_static/css/mixxx.css

```diff
@@ -157,6 +157,8 @@ a:hover {
 }
 .rst-content .guilabel {
     background: #faf2e7;
+    white-space: nowrap !important;
```

Comment:
Is `!important` really necessary?

Reply:
not really, would remove that now, was checking why `white-space: nowrap` didn't work at first; didn't commit my work.
I'll remove this now